ui/components/ui/editable-label/editable-label.js

```diff
@@ -73,12 +81,21 @@ class EditableLabel extends Component {
   }
 
   render() {
-    const { isEditing } = this.state;
-    const { className } = this.props;
+    const { isEditing, value } = this.state;
+    const { className, accountsNames } = this.props;
 
     return (
-      <div className={classnames('editable-label', className)}>
-        {isEditing ? this.renderEditing() : this.renderReadonly()}
+      <div>
+        <div className={classnames('editable-label', className)}>
+          {isEditing ? this.renderEditing() : this.renderReadonly()}
+        </div>
+        {accountsNames.includes(value) ? (
+          <div
+            className={classnames('send-v2__error', 'send-v2__error-amount')}
```

Comment:
These classes were written for the "Send" page, and should not be used here. The styles for this component should be under the `editable-label` namespace.

Reply:
Fixed in: d69e6682ef8cce5a6f0d7dd860acabac7ec614e3